src/containers/Application/Part3.js

```diff
@@ -20,13 +29,24 @@ export default () => {
 
   const handleNavigation = async href => {
     await forceSave()
+    if (href === '/application/review') {
+      const newErrors = validate(application.questionnaire)
+      if (checkForError(newErrors)) {
+        window.alert('Please complete all required fields.')
+        return
+      }
```

Comment:
Maybe these can be refactored.

Reply:
They exist in 2 other files

Reply:
I'm going to address this in a separate issue #228 (don't want to change it then have something break)